src/services/activities.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * name : services/activities.js
+ * author : Priyanka Pradeep
+ * Date : 13-Aug-2024
+ * Description : Activities Service
+ */
+const httpStatusCode = require('@generics/http-status')
+const activitiesQueries = require('@database/queries/activities')
+const responses = require('@helpers/responses')
+const { activityDTO } = require('@dtos/activity')
+const common = require('@constants/common')
+module.exports = class ActivityHelper {
+	/**
+	 * list activities.
+	 * @method
+	 * @name list
+	 * @param {Integer} resourceId - The ID of the resource for which activities are listed.
+	 * @param {String} userId - Id of the logged in user
+	 * @param {Integer} page - Page number
+	 * @param {Integer} page - Page size limit
+	 * @param {String} search - Search text.
+	 * @returns {JSON} - activities list response.
+	 */
+
+	static async list(resourceId, userId, orgId, page, limit) {
```

Comment:
@priyanka-TL where do we check if object id is not null?

Reply:
@aks30 In validator we are checking

Reply:
Added in service level too